contracts/DxAvatar.sol

```diff
@@ -0,0 +1,11 @@
+pragma solidity ^0.4.24;
```

Comment:
how about 0.0.25 as in arc ?

Reply:
I thought arc was `0.4.24`. Anyways, we 0.4.25 there's a problem. Some projects are not yet migrated to truffle 5, because as you know is a big change and tests and JS code must be adapted. 

So the solution to be able to have both versions of truffle working a the same time with the same contracts we need to use 0.4.24 (last version used by truffle 4)

For the ones in the DAO we could use 0.4.25, cause we use truffle 5, but I wonder if this would be a limitation while people is not fully migrated. What do you think?
Is there something important in 0.4.25 that would make it worth it?

Reply:
arc use 0.4.25.
in order to be able to verify the bytecode (against arc) it need to be compile with the same compiler version.

Reply:
Yep, I understand. But, I mean, is not wise to consider using `0.4.24` since has more compatibility.

We can adapt, it's just, we could weight the pros/cons

Reply:
Actually, I think we use the 0.4.25 compiler, but the pragma 0.4.24. Is not this a fare setup?

Reply:
It doesn't solve the issue, does it? 
I think https://www.npmjs.com/package/truffle/v/4.1.14 uses 0.4.24 compiler. I understand that if we try to compile a contract with pragma 0.4.25 will fail, right?

Or am I understanding it wrong?

Reply:
use truffle https://github.com/trufflesuite/truffle/releases/tag/v5.0.0-beta.2